### PR TITLE
Add variations on iml[i,e]ment->implement

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -9065,9 +9065,18 @@ imigrated->emigrated, immigrated,
 imigration->emigration, immigration,
 imilar->similar
 iminent->eminent, imminent, immanent,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I know it's not your change, but can we swap this so imminent is the first suggestion, given it's only one letter missing.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9065,9 +9065,18 @@ imigrated->emigrated, immigrated,
 imigration->emigration, immigration,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Likewise here, I assume they've got their e/i right and more likely to have missed a letter.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 0ce5399

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 0ce5399

**Pull request merged**